src/app/core/data/data.service.ts

```diff
@@ -90,4 +101,15 @@ export abstract class DataService<TNormalized extends CacheableObject, TDomain>
     // return this.rdbService.buildSingle(href));
   }
 
+  findEmbedded<TEmbeddedNormalized extends CacheableObject, TEmbeddedDomain>(
```

Comment:
Yes you're right, but there isn't a browse endpoint for collections so this rest call doesn't work : 

[http://dspace7.4science.it/dspace-spring-rest/api/discover/browses/dateissued/collections?scope=7669c72a-3f2a-451f-a3b9-9210e7a4c02f](http://dspace7.4science.it/dspace-spring-rest/api/discover/browses/dateissued/collections?scope=7669c72a-3f2a-451f-a3b9-9210e7a4c02f)

The right way to retrieve collections inside a community is this call : 

[http://dspace7.4science.it/dspace-spring-rest/api/core/communities/7669c72a-3f2a-451f-a3b9-9210e7a4c02f/collections](http://dspace7.4science.it/dspace-spring-rest/api/core/communities/7669c72a-3f2a-451f-a3b9-9210e7a4c02f/collections)

That's why I did the new method